gauravsingh1281/Rentalog-Rent-Management-Web-App-For-Landlords
# fixed-missing-testimonials

# What this PR do?
fixed #66 , This PR fixes the missing testimonials section.

# Step to verify
- go to home page
- scroll down 
- after the About us section you will see the testimonials section

# Additional info
![Screenshot from 2023-10-17 18-02-14](https://github.com/gauravsingh1281/Rentalog-Rent-Management-Web-App-For-Landlords/assets/129667408/bf319e7b-0f9f-4eb5-9769-65be730d9a17)
